qgis.json

```diff
@@ -5,15 +5,16 @@
         {
             "type" : "git",
             "url" : "https://github.com/kadas-albireo/QGIS.git",
-            "branch" : "kadas_modifications"
+            /* "branch" : "kadas_modifications", */
```

Comment:
I've seen comments break tooling once and again. You can make the key "//branch" and it should work.

Reply:
we can just remove it, instead. I have never had any problem, but flathub-extra-data-checker stripping them.

Reply:
I used an alternative, JSON compatible way to do it. The important point here is to indicate the branch for the maintainer, we can't use branch + commit nor a bare branch.